requirements.txt

```diff
@@ -8,6 +8,7 @@ colorama>=0.4.3
 cryptography==38.0.3
 dask==2022.9.2
 deepdiff==5.8.1
+expandvars==0.9.0
```

Comment:
nb. I originally used https://docs.python.org/3/library/os.path.html#os.path.expandvars, but it didn't do quite what I wanted, so I've added this library: https://pypi.org/project/expandvars/

See the pytest changes for a good picture of what this is used for.